tools/library/src/reference/initialize_reference_operations.cu

```diff
@@ -43,45 +43,47 @@ namespace cutlass {
+// void initialize_gemm_reference_operations_e5m2a_e4m3out(Manifest &manifest);
+// void initialize_gemm_reference_operations_e4m3a_e5m2out(Manifest &manifest);
+// void initialize_gemm_reference_operations_e5m2a_e5m2out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp8in_fp16out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp8in_bf16out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp8in_fp32out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp32out(Manifest &manifest);
+// void initialize_gemm_reference_operations_fp_other(Manifest &manifest);
+void initialize_gemm_reference_operations_fp_mixed_input(Manifest &manifest);
 
 void initialize_conv2d_reference_operations(Manifest &manifest);
 void initialize_conv3d_reference_operations(Manifest &manifest);
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
 void initialize_reference_operations(Manifest &manifest) {
-  initialize_conv2d_reference_operations(manifest);
-  initialize_conv3d_reference_operations(manifest);
-
-  initialize_gemm_reference_operations_int4(manifest);
-
-  initialize_gemm_reference_operations_int8_interleaved_32(manifest);
-  initialize_gemm_reference_operations_int8_interleaved_64(manifest);
-  initialize_gemm_reference_operations_int8_canonical(manifest);
-
-  initialize_gemm_reference_operations_e4m3a_e4m3out(manifest);
-  initialize_gemm_reference_operations_e5m2a_e4m3out(manifest);
-  initialize_gemm_reference_operations_e4m3a_e5m2out(manifest);
-  initialize_gemm_reference_operations_e5m2a_e5m2out(manifest);
-  initialize_gemm_reference_operations_fp8in_fp16out(manifest);
-  initialize_gemm_reference_operations_fp8in_bf16out(manifest);
-  initialize_gemm_reference_operations_fp8in_fp32out(manifest);
-
-  initialize_gemm_reference_operations_fp32out(manifest);
-  initialize_gemm_reference_operations_fp_other(manifest);
+  // initialize_conv2d_reference_operations(manifest);
+  // initialize_conv3d_reference_operations(manifest);
+
+  // initialize_gemm_reference_operations_int4(manifest);
+
+  // initialize_gemm_reference_operations_int8_interleaved_32(manifest);
+  // initialize_gemm_reference_operations_int8_interleaved_64(manifest);
+  // initialize_gemm_reference_operations_int8_canonical(manifest);
+
+  // initialize_gemm_reference_operations_e4m3a_e4m3out(manifest);
+  // initialize_gemm_reference_operations_e5m2a_e4m3out(manifest);
+  // initialize_gemm_reference_operations_e4m3a_e5m2out(manifest);
+  // initialize_gemm_reference_operations_e5m2a_e5m2out(manifest);
+  // initialize_gemm_reference_operations_fp8in_fp16out(manifest);
+  // initialize_gemm_reference_operations_fp8in_bf16out(manifest);
+  // initialize_gemm_reference_operations_fp8in_fp32out(manifest);
+
+  // initialize_gemm_reference_operations_fp32out(manifest);
+  // initialize_gemm_reference_operations_fp_other(manifest);
```

Comment:
RESOLVE THIS: Uncomment it before the merge. These are commented out for speed of development and reducing compilation times.